src/Network.h

```diff
@@ -110,7 +110,14 @@ class Network {
     static size_t get_hist_planes();
     static size_t get_num_output_policy();
 
+    static constexpr Move flip_move(Move move) {
+      // 07070 is the bit mask for the ranks (in octal), inverting those bits flips the board.
+      return Move(move ^ 07070);
```

Comment:
Hrmm, is this the same as the previous logic? (I would have guessed that the previous logic actually rotated the board, not flipped it...)  If not then this would invalidate the current trained state of the network.

(Certainly flipping the board seems more correct at first glance - but if its not actually whats already happening, I don't think this change can't be done like this... it needs to be a much more involved process?)

Reply:
Ahh never mind it appears the ~ operator is overloaded for square in ways which scare me. :P
I guess this might be okay... :)